### DISCUSSION TITLE
render page by user login

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7379] [Help]
I want to render pages based on the users login
as like if user is login then render some pages and if admin is login render some another pages

**Comment by R2bEEaton** [comment]
Sounds like what you want is some RBAC (role-based access control). Here's a [really long blog post](https://budibase.com/blog/app-building/role-based-access-control/) to learn more about this.

### For a quicker implementation, create a screen then set the access using the GUI.
![image](https://user-images.githubusercontent.com/34921506/185898852-2f3660cf-dd8a-4776-b211-f97efa5c1740.png)

### Or, to change the access of an existing screen, select the screen then then "Screens" icon on the top left.
![image](https://user-images.githubusercontent.com/34921506/185899142-b6c90250-0bef-4664-8991-1e49bf92d777.png)

As underlined above, you can also set a Home Screen for each role which will show first after logging in.
Hope this helps!

**Reply by melohagan** [reply]
There's a little bit here in the docs as well: https://docs.budibase.com/docs/screens#access-level

**Reply by DevrayePatil** [reply]
I want to create two different users like a service provider and the user for that service

**Reply by melohagan** [reply]
Like SaaS? There's a few discussion posts talking about that, including this one here: https://github.com/Budibase/budibase/discussions/3866
